scripts/setup-volume-testnet.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+if ! which jq > /dev/null; then
+  echo 'command jq not found, please install jq'
+  exit 1
+fi
+
+CHAIN_ID="paloma"
+
+palomad init my_validator --chain-id "$CHAIN_ID"
+
+pushd ~/.paloma/config/
+sed -i 's/^keyring-backend = ".*"/keyring-backend = "test"/' client.toml
+sed -i 's/^minimum-gas-prices = ".*"/minimum-gas-prices = "0.001grain"/' app.toml
+sed -i 's/^laddr = ".*:26657"/laddr = "tcp:\/\/0.0.0.0:26657"/' config.toml
+jq ".chain_id = \"${CHAIN_ID}\"" genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+jq 'walk(if type == "string" and .. == "stake" then "grain" else . end)' genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+popd
+
+VALIDATOR_STAKE_AMOUNT=100000000grain
+init_validator() {
+  name="$1"
+  address="$2"
+  
+  palomad add-genesis-account "$address" 250000000grain
+  palomad gentx "$name" "$VALIDATOR_STAKE_AMOUNT" --chain-id "$CHAIN_ID"
```

Comment:
@Vizualni Currently getting `Error: failed to fetch 'chase' from the keyring: chase.info: key not found` from this line, which makes sense. How do I add external keys without importing the private key?

Reply:
This makes sense. I think the "chase" validator will have to execute this line and I'll make a new one every script invocation.